project/profiles/capacity4more/modules/c4m/user/c4m_user/c4m_user.module

```diff
@@ -1286,3 +1299,51 @@ function c4m_user_ckeditor_settings_alter(&$settings, $conf) {
     unset($settings['loadPlugins']['media']);
   }
 }
+
+/**
+ * Implements hook_action_info_alter().
+ *
+ * Removes 'contact multiple' action, if user is not admin / group owner.
+ */
+function c4m_user_action_info_alter(&$actions) {
+  global $user;
+
+  if ($user->uid == 1) {
```

Comment:
I believe the meaning was the GA and not the SA, hence you probably want to use the `og_user_access()` here